### ISSUE TITLE
second boss in Shadowloabyrinth "Blackheart the Inciter"

### ISSUE THREAD

**Issue by Inyu** [Helios-Empire/Helios-Tracker#146]
he resets while he used his mindcontroll
i frapsed it gonna upload tomorrow
https://www.youtube.com/watch?v=UG9f0ZF2PyE
here is the vid

**Comment by HEGladios** [comment]
I have made some changes to the script. We will need some help testing this in the next few days.
Keep your eyes on the forums for details.

**Comment by Inyu** [comment]
another problem occurred which made buffs on partymember show as debuffs below their frame this stayed even after we relogged and made new party

**Comment by LoveYourBody** [comment]
Might this be a glitch in "SPELL_INCITE_CHAOS_B" incite chaos ? since it wasnt funcional untill you fixed mind control.

quote from the code: 
SDComment: Incite Chaos not functional since core lacks Mind Control support

**Comment by HEGladios** [comment]
I'm guessing he resets because all targets to friendly.
On retail he's just supposed to stand there and laugh until the 15 seconds are up.
I changed it so that he'd stay in combat during the Incite Chaos phase.

I'm not sure about the mind control debuff issue, that's another thing entirely.

**Comment by HEguinness** [comment]
Could you PM me the part of the code u changed, i wanna use the template of combat stuff for something

**Comment by Inyu** [comment]
There is still tiny timeframe where we get ooc (about ~0.5 sec) where he resets again
we avoided that with pets. They didn't get ooc and the fight resumed normally,
About the debuff issues:
In the mc phase everyone is hostile and the buffs are shown as debuffs, but even after the mc where ppl are friendly again the buffs are still shown as debuffs this stays all the time and is only limited to ppl who did the boss together. (e.g we did it yday with 2 ppl where this already was the case and 2 ppl where the display of the buffs/debuffs where normal after a mc phase everyone had their buffs shown as debuffs on the target frame)
this is only limited to the target and target target frame if i target myself the buffs are shown under the player portrait are normal but on the target frame they are marked as debuffs.

Hope this give a better insight into this issue.
